src/components/CourseCaution.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="course-outerWrapper course-tooltip">
+    <div v-if="!compact" class="course-iconWrapper course-iconWrapper--caution">
+      <img class="course-icon course-icon--caution" src="../assets/images/caution.svg" />
+    </div>
+    <div class="course-tooltiptext course-tooltiptext--caution" v-html="cautionString"></div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
+  props: {
+    compact: Boolean,
+    cautionString: String,
+  },
+});
+</script>
+
+<style scoped lang="scss">
+@import '@/assets/scss/_variables.scss';
+
+.course {
+  &-iconWrapper {
+    font-style: normal;
+    display: flex;
+    margin-left: 0.2rem;
+    align-items: center;
+
+    // TODO: styling for info icon on course card
```

Comment:
We don't have them on the site right now, but we did have info icons in the design, which I think is what this CSS styling is leftover from. Do we want it in this new component or back in Course?

Reply:
I will leave this here. Not sure how in the future whether we will be reusing the code in this component or start new.